src/lib/autoinstall/widgets/storage/partition_usage_tab.rb

```diff
@@ -0,0 +1,173 @@
+# Copyright (c) [2020] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "cwm/tabs"
+require "cwm/replace_point"
+require "cwm/common_widgets"
+require "autoinstall/widgets/storage/filesystem_attrs"
+require "autoinstall/widgets/storage/raid_attrs"
+require "autoinstall/widgets/storage/lvm_pv_attrs"
+require "autoinstall/widgets/storage/lvm_partition_attrs"
+require "autoinstall/widgets/storage/encryption_attrs"
+require "autoinstall/widgets/storage/size_selector"
+require "autoinstall/widgets/storage/used_as"
+
+module Y2Autoinstallation
+  module Widgets
+    module Storage
+      # Tab to manage the partition section options that depends on its usage (file system, LVM PV,
+      # RAID member, etc).
+      class PartitionUsageTab < ::CWM::Tab
+        # Constructor
+        #
+        # @param partition [Presenters::Partition] presenter for a partition section of the profile
+        def initialize(partition)
+          textdomain "autoinst"
+
+          @partition = partition
+          self.handle_all_events = true
+        end
+
+        # @macro seeAbstractWidget
+        def label
+          # TRANSLATORS: name of the tab to display the partition section options based on its usage
+          _("Used as")
```

Comment:
I guess it should be "Used As" (not 100% sure).

Reply:
[The Capitalizer](https://thecapitalizer.com/) agrees with you.

In other words, is "Used As" for all of English capitalization styles